website/content/docs/commands/plugin/info.mdx

```diff
@@ -9,6 +9,12 @@ description: The "plugin info" command displays information about a plugin in th
 The `plugin info` displays information about a plugin in the catalog.
 The plugin's type of "auth", "database", or "secret" must be included.
 
+## deprecation_status field
+
+As of 1.12, all builtin plugins will have an associated Deprecation
+Status. This status will be reflected in the `deprecation_status` key/value
+pair, seen below.  Non-builtin plugins will omit this field.
```

Comment:
Ah, `Non-builtin plugins will omit this field.` answers my comment above. If this is the direction we're going, I have no qualms with it.